### PR TITLE
added "stäng" and "Tyck till igen" buttons. "stäng" close the tool-wi…

### PR DESCRIPTION
**Pull request by hitomin21**

### CODE REVIEW

**Review by jacobwod — CHANGES_REQUESTED** [review]
Please use English in the code (whenever it's naming objects, functions, comments etc).

File: new-client/src/plugins/collector/collector.js [hunk 1]
```diff
@@ -28,6 +28,8 @@ class Collector extends React.Component {
         serviceConfig: this.serviceConfig
       }
     });
+
+    this.collectorModel.app = props.app;
```
Comment (jacobwod) [inline new-client/src/plugins/collector/collector.js:32]: What's the point of this? You already send the app as prop when model is instantiated (line 23). 
```suggestion
// Remove line
```

File: new-client/src/plugins/collector/components/Page.js [hunk 2]
```diff
@@ -573,8 +573,47 @@ class Page extends Component {
...
+        variant="outlined"
+        color="primary"
+        className={classes.buttonRight}
+        onClick={() => {
+          this.props.model.observer.publish("abort");
+        }}
+      >
+        Tyck till igen!
+      </Button>
+    );
```
Comment (jacobwod) [inline new-client/src/plugins/collector/components/Page.js:587]: `collectAgainButton` and `okButton` are virtually the same, except for the label.

Just use one of them, read the value of `this.props.options.collectAgain` inside it and set label accordingly.

**Review by jacobwod — CHANGES_REQUESTED** [review]
See comments in files.

File: new-client/src/plugins/collector/collector.js [hunk 3]
```diff
@@ -28,8 +28,7 @@ class Collector extends React.Component {
         serviceConfig: this.serviceConfig
       }
     });
-
-    this.collectorModel.app = props.app;
+// Remove line
```
Comment (jacobwod) [inline new-client/src/plugins/collector/collector.js:31]:
```suggestion
```

**Review by jacobwod — CHANGES_REQUESTED** [review]
The idea was that you actually remove the line, not add my comment… 😄

File: new-client/src/plugins/collector/collector.js (hunk 3 above)
Comment (hitomin21) [reply new-client/src/plugins/collector/collector.js:31]: I noticed and fixed it in another commit :)

File: new-client/src/plugins/collector/components/Page.js [hunk 4]
```diff
@@ -568,13 +573,39 @@ class Page extends Component {
         onClick={() => {
           this.props.model.observer.publish("abort");
         }}
+      >
+        {okButtonText}
```
Comment (jacobwod) [inline new-client/src/plugins/collector/components/Page.js:577]: Drop the `okButtonText` variable and just do a conditional render like this:
`this.props.options.collectAgain ? "Tyck till igen : "Stäng"`

**Review by jacobwod — CHANGES_REQUESTED** [review]

**Review by jacobwod — APPROVED** [review]

**Pull request merged**